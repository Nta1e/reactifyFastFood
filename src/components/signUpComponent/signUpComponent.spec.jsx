import React from 'react';
import { shallow } from 'enzyme';
import FormPage from './signUpComponent';

const props = {
    error: '',
    errorClass: '',
    onChange: jest.fn(),
    onSubmit: jest.fn(),
    success: '',
    successClass: ''
};
describe('test signup component',()=>{
    let wrapper;
    beforeEach(()=>{
        wrapper = shallow(<FormPage {...props} />)
    });
    it('matches snapshot', ()=> {
        expect(wrapper).toMatchSnapshot()
    })
});