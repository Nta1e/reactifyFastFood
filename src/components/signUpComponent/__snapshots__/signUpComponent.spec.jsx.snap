// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test signup component matches snapshot 1`] = `ShallowWrapper {}`;
